src/modules/rkhunter.rs

```diff
@@ -0,0 +1,172 @@
+//! Pretend to install composer packages
+use async_trait::async_trait;
+use chrono::Utc;
+use rand::prelude::*;
+use yansi::Paint;
+
+use crate::args::AppConfig;
+use crate::data::RKHUNTER_CHECKS_LIST;
+use crate::data::RKHUNTER_CHECKS_SHORT_LIST;
+use crate::data::RKHUNTER_INFOS_LIST;
+use crate::data::RKHUNTER_ROOTKITS_LIST;
+use crate::data::RKHUNTER_TASKS_LIST;
```

Comment:
Could you merge these?

Reply:
Reduced it to RKHUNTER_CHECKS_LIST, RKHUNTER_ROOTKITS_LIST and RKHUNTER_TASKS_LIST.